grammars/css.cson

```diff
@@ -361,11 +361,11 @@
       }
       {
         # @keyframes
-        'begin': '(?i)(?=@keyframes([\\s\'"{;]|/\\*|$))'
+        'begin': '(?i)(?=@(?:(?:-(?:webkit|moz|o|ms)-)?keyframes)([\\s\'"{;]|/\\*|$))'
```

Comment:
The outside capture can be removed, so this can be simplified to be `(?:-(?:webkit|moz|o|ms)-)?keyframes`.  Same for below.

`(?i)(?=@(?:-(?:webkit|moz|o|ms)-)?keyframes([\\s\'"{;]|/\\*|$))`